engine/src/main/java/org/entando/entando/aps/system/services/component/ComponentService.java

```diff
@@ -189,4 +196,30 @@ private boolean checkReferences(List<ComponentDeleteRequestRow> componentsToDele
     static class ComponentDeletionStep {
         private boolean successful;
     }
+    
+    public static class ComponentDeleteRequestRowComparator implements Comparator<ComponentDeleteRequestRow> {
```

Comment:
You should put this inner class inside `ComponentDeleteRequestRow` not in a service, usually we don't put an inner class about DTO in a service, this comparator could be used outside this service.

Reply:
Done